src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,45 @@
+# selector-max-specificity
+
+Limit the specificity of selectors.
+
+````
+.foo, #bar.baz span, #hoo { color: pink; }
+/** ↑         ↑        ↑
+ * These selectors */
+````
+
+# Options
+
+`string`: Level of specificity allowed
+
+Format is id,class,type as computed by the [W3C specificity calculator](https://www.w3.org/TR/selectors/#specificity)
+
+For a visual representation of selector specificity, visit: [https://specificity.keegan.st](https://specificity.keegan.st)
+
+For example, with "0,2,0":
+
+The following patterns are considered warnings:
+
+````
+.thing .thing2 .thing3 {}
+````
+
+````
+.thing .thing2 {
+  .thing3 & {}
+}
+````
+
+The following are __not__ considered warnings:
+
+````
+.thing div {}
+````
+
+````
+.thing div {
+  a {}
+}
+````
+
+**Note**: currently [direct nesting](http://tabatkins.github.io/specs/css-nesting/#direct) is supported but the [@nest](http://tabatkins.github.io/specs/css-nesting/#at-nest) at-rule is not.
```

Comment:
Also, I removed this section of the README.md already in https://github.com/benfrain/stylelint/commit/524040aa038e04ff214a766ece12f968126bf10f